ur_bringup/launch/ur_moveit.launch.py

```diff
@@ -41,6 +41,7 @@ def launch_setup(context, *args, **kwargs):
     use_fake_hardware = LaunchConfiguration("use_fake_hardware")
     fake_sensor_commands = LaunchConfiguration("fake_sensor_commands")
     launch_rviz = LaunchConfiguration("launch_rviz")
+    launch_rviz = LaunchConfiguration("launch_servo")
```

Comment:
You probably meant something like this:
```suggestion
    launch_servo = LaunchConfiguration("launch_servo")
```